src/content/app/entity-viewer/variant-view/variant-view-sidebar/VariantViewSideBar.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { useAppSelector } from 'src/store';
+
+import Sidebar from 'src/shared/components/layout/sidebar/Sidebar';
+
+import { getEntityViewerSidebarTabName } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors';
```

Comment:
Should we even bother with sidebar tabs at this stage? I remember Andrea said that she wasn't sure she will have use for the second tab (the one that says alternative IDs and locations in XD).

Reply:
I think we should leave it. Andrea said there will be a second tab. She also said to create a dummy tab. I will see if I can add it to this PR. If not, its ok not a blocker